### DISCUSSION TITLE
🛜 T-Pot Mobile - The All In One Wireless Honeypot Solution

### DISCUSSION THREAD

**Discussion by t3chn0m4g3** [telekom-security/tpotce#1550] [Announcements]
🎉 Just completed open sourcing ...
# 🛜 [T-Pot Mobile](https://github.com/telekom-security/tpotmobile) - The All In One Wireless Honeypot Solution
🍯 T-Pot Mobile is powered by a Raspberry Pi 4, features a 4.3" touch display, housed in a self-designed 3D-printed case
🔋 The integrated battery pack makes it self-sufficient for 8+ hours and thus a great companion on the go for educational purposes, presentations, or hands-on demonstrations of cybersecurity in action
✅ Everything is now readily available as open source, powered by [Telekom Security](https://github.com/telekom-security) here at [GitHub](https://github.com/telekom-security/tpotmobile)
⚙️ Just add the hardware and you are ready to go
😍 Looking forward to seeing your 3d prints, spins and forks! Sharing is caring!

![image](https://github.com/telekom-security/tpotce/assets/4318452/f83e067b-79d8-4a4e-96bd-4fb1f2f254fc)

**Comment by lissy93** [comment]
Looks awesome!
I love how clear and detailed the docs in the readme are :)

**Reply by t3chn0m4g3** [reply]
Thank you 😊

**Comment by darkanubis0100** [comment]
Wow!

Nice gadget 👌

**Reply by t3chn0m4g3** [reply]
Thank you ☺️

**Comment by RyanDoyon** [comment]
What a fun way to integrate the already great TPot platform! Guess it's time I finally buy a 3D printer.

**Reply by t3chn0m4g3** [reply]
Thank you! 3d printing is so much fun 🤩

**Comment by ghenry** [comment]
Awesome job! Congratulations!

**Reply by t3chn0m4g3** [reply]
Thank you, Gavin 🤗
